webview/platforms/cocoa.py

```diff
@@ -635,7 +638,7 @@ def dummy_completion_handler():
 
         config.preferences().setValue_forKey_(webview_settings['ALLOW_FILE_URLS'], 'allowFileAccessFromFileURLs')
 
-        if _state['debug'] and webview_settings['OPEN_DEVTOOLS_IN_DEBUG']:
+        if _state['debug']:
```

Comment:
Drive-by change, split into a separate commit if we want to drop: I don't think we should be checking `OPEN_DEVTOOLS_IN_DEBUG` here, since right now disabling `OPEN_DEVTOOLS_IN_DEBUG` means there's no devtools shown at all.